landing_page/templates/landing_page/base.html

```diff
@@ -74,7 +74,7 @@ <h1>ReeldIn</h1>
     <a href="{% url 'landing_page:index' %}"> Home</a>
     <a href="{% url 'landing_page:about' %}">About</a>
     <!-- remove movie tab from nav later, testing only-->
-    <a href="{% url 'landing_page:movie' %}">movie</a>
+    <a href="{% url 'landing_page:movie' movie_id=1 %}">movie</a>
```

Comment:
this doesn't work :(

Reply:
Oh yeah I didn't add any logic to the page except the name, that's what Elizabeth is doing. I just did this so it wouldn't throw an error

Reply:
Wait I just saw that it gives an error because there's no movie with id of 1